kythe/cxx/indexer/cxx/GraphObserver.h

```diff
@@ -761,6 +764,12 @@ class GraphObserver {
   virtual void recordTypeEdge(const NodeId& TermNodeId,
                               const NodeId& TypeNodeId) {}
 
+  /// \brief Records the type of a node's initializer as an edge in the graph.
```

Comment:
Which one? A node may have several initializers, e.g.

```
class A {
  A(): mem_(AnotherOne()) {}
  A(double yano): mem_(yano) {} 

  int mem_ = Default()l
};
```

Each of those initializations of `mem_` may be from a differently typed result, but using the semantic node has no way of distinguishing them.